File: web/src/app/shared/Models/question.ts
export class Question {
    QnId: number = 0;
    QuizId: number = 0;
    Qn: string='';
    ImageName: string='';
    Option1: string='';
    Option2: string='';
    Option3: string='';
    Option4: string='';
    Answer: string='';
}

export class QuestionModel {
    qnID: number = 0;
    quizID: number = 0;
    qn: string='';
    imageName: string='';
    option1: string='';
    option2: string='';
    option3: string='';
    option4: string='';
    answer: string='';
}

export class QuestionAnswer {
    stt: number = 0;
    qnid: number = 0;
    quizid: number = 0;
    qn: string='';
    imagename: string='';
    options: string[]=[];
    answer: string='';
    selected: string='';
}

File: E-LearnignWebAPI/ElearningBO/E-Learning/Question.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ElearningBO.E_Learning
{
    public class Question
    {
        [Key]
        public int QnID { get; set; }

        [ForeignKey("Quiz")]
        public int QuizID { get; set; }

        public string Qn { get; set; }
        public string ImageName { get; set; }
        public string Option1 { get; set; }
        public string Option2 { get; set; }
        public string Option3 { get; set; }
        public string Option4 { get; set; }
        public string Answer { get; set; }
        public bool isDelete { get; set; }
        public virtual Quiz Quiz { get; set; }
    }

    public class QuestionModelExcel
    {
        public int QuizID { get; set; }
        public string Qn { get; set; }
        public string ImageName { get; set; }
        public string Option1 { get; set; }
        public string Option2 { get; set; }
        public string Option3 { get; set; }
        public string Option4 { get; set; }
        public string Answer { get; set; }
    }
}
